_api-files/embedding-api/objects/auth-url-parameters.md

```diff
@@ -0,0 +1,32 @@
+---
+content-type: "embed-url-parms"
+
+parameters:
+  - name: "client_id"
+    required: true
+    description: |
+      Your application's client ID. This is obtained when you register your application with [{{ page.contact-email }}](mailto: {{ page.contact-email }}).
+
+  - name: "redirect_uri"
+    required: false
+    description: |
+      The callback URL for your application, which will be used in [Step 3](#authentication--callback-to-app).
+
+      **Note**: If provided in Step 1 as a URL parameter, then the value must match one of the redirect URIs provided with your application's registration.
```

Comment:
Also note: If not provided in Step 1, Stitch will fallback to the first redirect_uri associated with the API client's account.